dashboard/src/pages/TreeDetails/TreeDetails.tsx

```diff
@@ -144,13 +144,16 @@ function TreeDetails(): JSX.Element {
           date: value.start_time,
           buildTime: value.duration,
           compiler: value.compiler,
-          buildErrors: value.valid ? 0 : 1,
-          status: value.valid ? 'valid' : 'invalid',
+          buildErrors: value.valid || value.valid === null ? 0 : 1,
```

Comment:
can you wrap this logic into a function called `isBuildError` so it makes a little more clear there as reason for this madness ?